### PR TITLE
Car model

### PR DESCRIPTION
**Pull request by Arturgouveia1970**
In this PR:
- Created a car model.
- Created car controller.

### CODE REVIEW

File: spec/models/car_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
```
Comment (ppmarq1) [inline spec/models/car_spec.rb:2]: (optional) I reccomend to add spec for cars.

**Review by ppmarq1 — COMMENTED** [review]
Hello @Arturgouveia1970 good job! You are almost there! Please just add a spec for cars.

Pedro

**Comment by Arturgouveia1970** [comment]
Hello reviewer, I do not understand why my PR was not aproved. The spec is not a project requirement for this exercise, at least is not mentioned in the kanban board card. Please advise me on how to proceed. ArturEnviado do meu telemóvel Huawei.-------- Mensagem original --------De: Pedro Marques ***@***.***>Data: quinta-feira, 10 de novembro de 2022, 18:06Para: Arturgouveia1970/car-rental-backend ***@***.***>Cc: Artur Gouveia ***@***.***>, Mention ***@***.***>Assunto: Re: [Arturgouveia1970/car-rental-backend] Car model (PR #16)
@ppmarq1 commented on this pull request.

Hello @Arturgouveia1970 good job! You are almost there! Please just add a spec for cars.
Pedro

In spec/models/car_spec.rb:
> @@ -0,0 +1,5 @@
+require 'rails_helper'
+

Please add spec for cars

**Comment by ppmarq1** [comment]
Hello @Arturgouveia1970 . You are right, this is not a requirement, I am moving this as Optional. But I highly recommend doing it as a good practice for developers. Rspec tests can minimize bugs and guarantee a good code structure.
This PR was approved

**Review by ppmarq1 — APPROVED** [review]
Approved. Now it is time to merge it!

**Comment by Arturgouveia1970** [comment]
hello @ppmarq1, true, but when we do the rswag it creates tests for the api.

**Pull request merged**